riposte-spi/src/main/java/com/nike/riposte/server/http/RequestInfo.java

```diff
@@ -384,4 +384,15 @@ public default String getPathParam(String key) {
      * do not need to worry about this issue - it's a problem for the server to solve.
      */
     public void releaseMultipartData();
+
+    /**
+     * This will return the path template set by {@link #setPathParamsBasedOnPathTemplate(String)} .
```

Comment:
Couple quick things on this comment:

1. Paragraphs separated by whitespace won't render as paragraphs unless you put in HTML to make them render that way. The best way I've found that still looks good when looking at source code, doesn't get in the way too much, and doesn't cause javadoc warnings/errors is to put a `<p>` tag at the start of the paragraph (note - not a self-closed `<p/>` tag as that will cause javadoc warnings/errors when building in gradle/maven/etc). There are older comments in this codebase that don't follow this convention - don't bother fixing old stuff, but seems like we might as well do it for new comments.
2. Include the regular path so that people can see the difference between the template and full path.

The comment with both of these suggestions and a little bit of extra clarification might look like:

``` java
    /**
     * This will return the path template set by {@link #setPathParamsBasedOnPathTemplate(String)}, which corresponds to an endpoint {@link Matcher}'s path template and is automatically set when a request is matched to an endpoint.
     *
     * <p>Example path template: "/app/{appId}/user/{userId}". Note this different than {@link #getPath()}, which in this example might be "/app/foo/user/bar".
     *
     * <p>Will never be null - an empty string will be used if {@link #setPathParamsBasedOnPathTemplate(String)}
     * was never called (should only happen in error cases where an endpoint was not called).
     */
```

I'd also update the `getUri()` and `getPath()` comments to point people to the new `getPathTemplate()` method if they're looking for the template instead of the full raw URI or path.